src/bundler.ts

```diff
@@ -175,32 +194,41 @@ export class Bundler {
                     filePath: imp.fullPath,
                     tilde: imp.tilde,
                     found: true,
-                    imports: childImports
+                    imports: childImports,

```

Comment:
Remove unnecessary comma.

Reply:
Sorry its a habit for me in my stuff to include trailing comma. Removing